src/sentry/integrations/aws_lambda/utils.py

```diff
@@ -36,6 +44,26 @@ def get_arn_without_version(arn):
     return arn.rpartition(":")[0]
 
 
+def get_version_of_arn(arn):
```

Comment:
@scefali do you think we should have some unit tests for these utility functions? Test the happy path and some bad paths.

Also should we raise an exception if not `arn string` is passed?

Reply:
@NisanthanNanthakumar Hmmm...the spec of these utility functions will get tested in some of the other tests. I can still add unit tests specifically for these util functions if you think the redundancy makes sense.

>Also should we raise an exception if not valid arn string is passed?

We would raise an exception if we can't get the integer off the end the ARN which is what we want. I don't think we need to test for the rest of the ARN (other functions will blow up if other parts of the ARN are messed up.

Reply:
@scefali hmm I think it best to have specific tests for each function bc in the future, we might refactor the functions where we call these functions add remove them and use them elsewhere. We can't assume those tests for the calling functions will always be there and will be testing these utility functions.

Reply:
@NisanthanNanthakumar good point! I'll add some unit tests

Reply:
@NisanthanNanthakumar tests added